modules/statics/src/coins.ts

```diff
@@ -7502,6 +7502,70 @@ export const coins = CoinMap.fromCoins([
     '0xc770eefad204b5180df6a14ee197d99d808ee52d',
     UnderlyingAsset.FOX
   ),
+  erc20(
+    '523d92b8-5fc8-4d13-8359-b69f3ac64382',
+    'cube',
+    'CUBE',
+    18,
+    '0x622dffcc4e83c64ba959530a5a5580687a57581b',
+    UnderlyingAsset.CUBE
+  ),
+  erc20(
+    'ae798fd0-30cc-47de-af7d-cae0e3db15ad',
+    'xx',
+    'XX',
+    8,
+    '0x47f33feb70ec153fc22755908a4061777d22c2da',
+    UnderlyingAsset.XX
+  ),
+  erc20(
+    'd340bbfc-1215-418d-8e57-64ffc954e2d7',
+    'litt',
+    'LITT',
+    18,
+    '0xc5b3d3231001a776123194cf1290068e8b0c783b',
+    UnderlyingAsset.LITT
+  ),
```

Comment:
Are we sure is LITT and not LIT?
https://etherscan.io/address/0xc5b3d3231001a776123194cf1290068e8b0c783b

Reply:
We already have something called `LIT` - https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts#L4100, which is different from this one

For lack of a better name, I added `LITT` but I'm open to suggestions